src/main/scala/org/broadinstitute/dsde/workbench/sam/config/package.scala

```diff
@@ -22,20 +27,26 @@ package object config {
     )
   }
 
-  implicit val resourceRoleReader: ValueReader[ResourceRole] = ValueReader.relative { config =>
-    ResourceRole(
-      ResourceRoleName(config.getString("roleName")),
-      config.as[Set[String]]("roleActions").map(ResourceAction)
-    )
+  def unquote(str: String): String = str.replace("\"", "")
```

Comment:
is this still needed? the object names don't have quotes around them anymore (but they did in a previous commit)

Reply:
yes -- for some reason, typsafe extra-quotes paths before passing them to `read()`